src/app/fdctl/run/tiles/fd_pack.c

```diff
@@ -540,19 +542,10 @@ during_frag( void * _ctx,
     fd_memcpy( TXN(ctx->cur_spot),     txn,     fd_txn_footprint( txn->instr_cnt, txn->addr_table_lookup_cnt ) );
     ctx->cur_spot->payload_sz = payload_sz;
   } else {
-    /* Here there is just a transaction payload, so it needs to be
-       parsed.  We can parse right out into the pack structure. */
     FD_MCNT_INC( PACK, GOSSIPED_VOTES_RECEIVED, 1UL );
```

Comment:
Move this metric to the dedup tile or poh. That'll involve changing the metrics.xml file and rerunning the python script to regenerate the headers. The metric is helpful, but now it's disabled, right?

Reply:
Will do.  It's just always 0 at the moment.  The second it turns 1 the program is going to terminate due to FD_LOG_ERR(), so it does make sense to move this elsewhere.